app/workers/deletes.php

```diff
@@ -122,6 +127,35 @@ public function shutdown(): void
     {
     }
 
+
+    /**
+     * @param int $timestamp
+     */
+    protected function deleteCache(int $timestamp): void
+    {
+        $this->deleteForProjectIds(function (string $projectId) use ($timestamp) {
+
+            $dbForProject = $this->getProjectDB($projectId);
+            $cache = new Local(APP_STORAGE_CACHE);
+
+            $this->deleteByGroup(
+                'cache',
+                [
+                new Query('accessedAt', Query::TYPE_LESSER, [$timestamp])
+                ],
+                $dbForProject,
+                function (Document $document) use ($cache) {
+                    $path = $cache->getRoot() . '/' . $document->getAttribute('path') . '/' .  $document->getId();
```

Comment:
Shouldn't we also delete the database log?

Reply:
We are using deleteByGroup()